addon.py

```
@@ -2148,17 +2216,34 @@ def create_hunyuan_job_local_site(
+                    if ssrf_err:
+                        return {"error": ssrf_err}
+                    # Pin to validated IP to prevent DNS rebinding
+                    parsed_img = urlparse(image)
+                    pinned_img_url = image.replace(
+                        f"{parsed_img.scheme}://{parsed_img.hostname}",
+                        f"{parsed_img.scheme}://{validated_ips[0]}",
+                        1,
+                    )
                     try:
-                        resImg = requests.get(image)
+                        resImg = requests.get(
```

Ruff (0.15.5)

[error] Probable use of `requests` call without timeout

(S113)